azure/packages/azure-client/package.json

```diff
@@ -33,7 +33,7 @@
 		"prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
 		"start:tinylicious:test": "npx @fluidframework/azure-local-service > tinylicious.log 2>&1",
 		"test:realsvc": "npm run test:realsvc:tinylicious",
-		"test:realsvc:local:run": "mocha --unhandled-rejections=strict --recursive dist/test/**/*.spec.js --exit --timeout 10000",
+		"test:realsvc:local:run": "mocha --unhandled-rejections=strict --recursive 'dist/test/**/*.spec.js' --exit --timeout 10000",
```

Comment:
Of the 5 or so packages I tried, only matrix repro'ed the issue...  but the added single quotes won't hurt.